src/ExpressionEvaluator/Core/Source/ExpressionCompiler/MetadataUtilities.cs

```diff
@@ -178,9 +175,24 @@ internal static string GetFileNameWithoutExtension(string fileName)
             return fileName;
         }
 
+        private static byte[] GetWindowsProxyBytes()
+        {
+            var assembly = typeof(ExpressionCompiler).GetTypeInfo().Assembly;
```

Comment:
GetExecutingAssembly is bad API. Should never be used. I don't think it's portable anyways.